numpy/core/fromnumeric.py

```diff
@@ -380,6 +380,9 @@ def choose(a, choices, out=None, mode='raise'):
     See Also
     --------
     ndarray.choose : equivalent method
+    numpy.take_along_axis : Take values from the input array by matching 1d index
+                            and data slices. May be preferable in the case that 'choices'
```

Comment:
Lines are too long, try to keep them <= 75 chars for documentation. Would be good to shorten the description to one line if possible, a link to the function will be generated and interested folks can click through.